container/docker/handler.go

```diff
@@ -135,18 +135,18 @@ func newDockerContainerHandler(
 	id := ContainerNameToDockerId(name)
 
 	// Add the Containers dir where the log files are stored.
-	otherStorageDir := path.Join(path.Dir(storageDriverDir), pathToContainersDir, id)
+	otherStorageDir := path.Join(storageDir, pathToContainersDir, id)
 
-	rwLayerID, err := getRwLayerID(id, storageDriverDir, dockerVersion)
+	rwLayerID, err := getRwLayerID(id, storageDir, storageDriver, dockerVersion)
```

Comment:
Yeah. That is possible. Any specific reason other than it being a no-op as of now?